FreshRSS/FreshRSS
# Improved: Subscription management with long lists

Rref #2459

# (Still a draft, feedback welcome)

The issue right now: the category boxes are very short. It is difficult to have a proper overview about all feeds, if the user has a lot feeds.

My idea:
- switch the view via button: from box-view to a column view with full extended lists


Before
![grafik](https://user-images.githubusercontent.com/1645099/193430221-ce5edcaf-b125-4057-8059-619e307c2530.png)

After
1) horizontal scrolling
2) fully expanded feed list
![grafik](https://user-images.githubusercontent.com/1645099/193430223-67157aef-1cde-4c9a-9743-a65799b5ade6.png)


Changes proposed in this pull request:

- tbd
-
-

How to test the feature manually:

1. go to "subscription management"
2. click on the button ↔️
3. see the "Trello like" column view


Pull request checklist:
